cmd/skopeo/main.go

```diff
@@ -92,7 +96,10 @@ func getPolicyContext(c *cli.Context) (*signature.PolicyContext, error) {
 	policyPath := c.GlobalString("policy")
 	var policy *signature.Policy // This could be cached across calls, if we had an application context.
 	var err error
-	if policyPath == "" {
+	if c.GlobalBool("policy-insecure-allow-anything") {
+		insecurepolicy := []byte(`{"default":[{"type":"insecureAcceptAnything"}]}`)
+		policy, err = signature.NewPolicyFromBytes(insecurepolicy)
```

Comment:
This would be more type-safe, and visibly break if we ever were unfortunate enough to change the format, as (roughly, untested)

```go
&Policy{
  Default: PolicyRequirements{NewPRInsecureAcceptAnything()}
}
```